[ao] fixing public v private for quantize.py

HDCharles opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #86037
* #86036
* #86034
* #86033
* #86032
* #86031
* #86030
* #86029
* #86028
* #86027
* #86026
* #86025
* #86024
* __->__ #86023
* #86022

Summary: just needed to add __all__

Test Plan: python test/test_public_bindings.py

Reviewers:

Subscribers:

Tasks:

Tags:

HDCharles committed:
[ao] fixing public v private for quantize.py

Summary: just needed to add __all__

Test Plan: python test/test_public_bindings.py

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 commented on torch/ao/quantization/quantize.py:
```diff
@@ -28,6 +28,24 @@
+    "quantize_qat",
+    "convert",
+    "swap_module",
+    "get_observer_dict",
```
feels like this function doesn't belong here, probably should put this in some util files I think

HDCharles committed:
Update on "[ao] fixing public v private for quantize.py"


Summary: just needed to add __all__

Test Plan: python test/test_public_bindings.py

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

HDCharles committed:
Update on "[ao] fixing public v private for quantize.py"


Summary: just needed to add __all__

Test Plan: python test/test_public_bindings.py

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

HDCharles commented:
@pytorchmergebot merge
